letsencrypt/cli.py

```diff
@@ -878,9 +963,11 @@ def prepare_and_parse_args(plugins, args):
                 "multiple -d flags or enter a comma separated list of domains "
                 "as a parameter.")
     helpful.add(
-        None, "--duplicate", dest="duplicate", action="store_true",
-        help="Allow getting a certificate that duplicates an existing one")
-
+        None, "--keep-until-expiring", "--keep", "--reinstall",
```

Comment:
One reason for the multiple names is to basically always have a name that matches the button the user presses ("Reinstall" for run mode, "Keep" for certonly); the other reason is to have a longform default that is fairly self-explanation, and a short abrreviation.  Maybe that can be -k.